<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserModifierWindow</class>
 <widget class="QMainWindow" name="UserModifierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>505</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modify a user's information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(244,244,244);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutTotal">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelUsername">
          <property name="styleSheet">
           <string notr="true">font-family:Microsoft Yahei;
font:22px;
color:#888888;</string>
          </property>
          <property name="text">
           <string>Username:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="usernameDisplay">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family:Microsoft Yahei;
font-size:22px;
font-weight:600;
padding-top:5px;
padding-bottom:5px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelPassword">
          <property name="styleSheet">
           <string notr="true">font-family:Microsoft Yahei;
font:22px;
color:#888888;</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QComboBox" name="cbxAuthority">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1.5px solid gray;
    color: rgba(51,51,51,1);
    background: white;
    text-align: AlignHCenter;
    background-color: white;
	font:22px;
	font-weight:600;
}

QComboBox QAbstractItemView {
    outline: 1px solid gray;
    border-radius:10px;
    color:rgb(123,123,123);
}
 
QComboBox QAbstractItemView::item {
    min-height: 15px;
	font-size:15px;
}
 
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255,128);
}
 
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
}
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft Yahei</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1.5px solid rgb(141,141,141);
font-family:Microsoft Yahei;
font:22px;
font-weight:600;
background-color:white;
padding-top:5px;
padding-bottom:5px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>376</width>
          <height>36</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnConfirm">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    font:17px;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:15px;
	font-weight:500;
    height:40px;
	padding-top:5px;
	padding-bottom:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
}</string>
          </property>
          <property name="text">
           <string>CONFIRM</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerBtnMiddle">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>7</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
